### ISSUE TITLE
Graphviz.org is inaccessible

### ISSUE THREAD

**Issue by badair** [ellson/graphviz#75]
I can ping it, but I can't browse the pages. I don't know how else to get in touch with the owner, so I thought I'd post here.

**Comment by emdenrg** [comment]
On 2/24/16 6:49 AM, Barrett Adair wrote:
>
> I can ping it, but I can't browse the pages. I don't know how else to 
> get in touch with the owner, so I thought I'd post here.
>
It gets very sluggish at times. It should be okay now.

     Emden

**Comment by graphviz** [comment]
Yes, I rebooted it.  We should just make a cron job to reboot it every night.
It’s funny that Linux has caught up to Windows.  Of course we should actually
get onto a better hosting environment and install patches regularly ha ha ha

**Comment by rohithmeethal** [comment]
Ahm still not accessible :-(

**Comment by ellson** [comment]
I can see http://graphviz.org/ from here without difficulty.   Is there any chance that the problem is at your end?

**Comment by ellson** [comment]
The IP address is: 204.178.9.49

**Comment by rohithmeethal** [comment]
Now it is resolved, may be just issue of that day. Thank you :)

**Comment by gregtampa** [comment]
Google Chrome could not load the webpage because www.graphviz.org took too long to respond. The website may be down, or you may be experiencing issues with your Internet connection.

**Comment by jf990** [comment]
Looks like some networking issue inside 12.251.32.6 
```
$ traceroute 204.178.9.49
traceroute to 204.178.9.49 (204.178.9.49), 64 hops max, 52 byte packets
---
 4  12.125.38.89 (12.125.38.89)  13.420 ms  7.265 ms  7.165 ms
 5  cr1.la2ca.ip.att.net (12.122.104.46)  74.955 ms  77.238 ms  75.555 ms
 6  cr1.slkut.ip.att.net (12.122.30.29)  80.271 ms  78.746 ms  82.218 ms
 7  cr2.dvmco.ip.att.net (12.122.30.26)  74.279 ms  74.824 ms  75.891 ms
 8  cr1.cgcil.ip.att.net (12.122.31.86)  77.945 ms  77.175 ms  79.633 ms
 9  cr2.cgcil.ip.att.net (12.122.2.54)  74.919 ms  77.723 ms  75.987 ms
10  cr1.n54ny.ip.att.net (12.122.1.1)  79.944 ms  76.234 ms  86.239 ms
11  cr81.nw2nj.ip.att.net (12.122.105.30)  79.991 ms  75.419 ms  75.859 ms
12  gar21.n54ny.ip.att.net (12.122.105.61)  115.446 ms  187.294 ms  91.660 ms
13  12.251.32.6 (12.251.32.6)  75.654 ms  75.559 ms  75.196 ms
14  * * *
15  * * *
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
```

**Comment by bytehala** [comment]
So it isn't just me then.

**Comment by graphviz** [comment]
I rebooted it just now. “It’s up for me.”

There’s also a persistent routing problem, that can easily be verified, e.g.
http://looking-glass.iprimus.net.au/lg/
The route dies when it hits AT&T 7018 (network 12). 
Similar problems from some places in Europe.  e.g. http://www.ntua.gr/cgi-bin/rtracert?name=www.graphviz.org
terminates without ever hitting our site - instead it goes off to some organization called cogentco.com 
We probably need to move our hosting and/or name service but it’s a project.

**Comment by bytehala** [comment]
Thanks. Don't know if this is helpful: I'm in Asia.

**Comment by GeospatialPython** [comment]
Graphviz.org inaccessible from the US 5/24/16 2200 GMT.
